### PR TITLE
Add coverage to private function Version.to_matchable/2

### PR DESCRIPTION
**Pull request by bjfish**
…_pre?)

### CODE REVIEW

**Comment by ericmj** [comment]
The test seems to fail, also try to keep your commit message headlines shorter.

**Comment by josevalim** [comment]
Nitpick: Elixir has no "methods" but "functions". :)

**Comment by bjfish** [comment]
Okay, I've fixed the test now.

**Comment by josevalim** [comment]
@bjfish can you also amend the commit message to use "function" instead of "method" and make it shorter? Thank you.

File: lib/elixir/test/elixir/version_test.exs [hunk 1]
```diff
@@ -111,6 +111,9 @@ defmodule VersionTest do
     assert V.match?("1.2.3-alpha", "1.2.3-alpha")
 
     assert V.match?("0.9.3", "== 0.9.3+dev")
+
+    {:ok, req} = Version.parse("2.3.0")
```
Comment (lexmag) [inline lib/elixir/test/elixir/version_test.exs]: It'd be better to rename `req` to `vsn`.
Requirement is a second argument for `match?` function, the first – version.

**Comment by bjfish** [comment]
@josevalim I amended the commit message.
@lexmag I fixed `req` to `vsn`

**Comment by josevalim** [comment]
@ericmj all yours. :)

**Comment by ericmj** [comment]
Thank you @bjfish!

**Pull request merged**